<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainDlgClass</class>
  <widget class="QMainWindow" name="MainDlgClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>549</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>400</width>
        <height>300</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>空当接龙计算器</string>
    </property>
    <property name="windowOpacity">
      <double>1.000000000000000</double>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>26</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>9</pointsize>
        </font>
      </property>
      <widget class="QMenu" name="menu">
        <property name="font">
          <font>
            <pointsize>9</pointsize>
          </font>
        </property>
        <property name="title">
          <string>游戏</string>
        </property>
        <addaction name="actionNewgame"/>
        <addaction name="actionCalculate"/>
        <addaction name="separator"/>
        <addaction name="actionPrev"/>
        <addaction name="actionNext"/>
        <addaction name="actionPlay"/>
      </widget>
      <widget class="QMenu" name="menu_2">
        <property name="title">
          <string>其他</string>
        </property>
        <addaction name="actionConfig"/>
      </widget>
      <addaction name="menu"/>
      <addaction name="menu_2"/>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <addaction name="actionNewgame"/>
      <addaction name="actionCalculate"/>
      <addaction name="actionPrev"/>
      <addaction name="actionNext"/>
      <addaction name="actionPlay"/>
      <addaction name="actionConfig"/>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <action name="actionNewgame">
      <property name="icon">
        <iconset resource="GlobalRes.qrc">
          <normaloff>:/Image/new.png</normaloff>:/Image/new.png
        </iconset>
      </property>
      <property name="text">
        <string>选择游戏</string>
      </property>
    </action>
    <action name="actionCalculate">
      <property name="icon">
        <iconset resource="GlobalRes.qrc">
          <normaloff>:/Image/calc.png</normaloff>:/Image/calc.png
        </iconset>
      </property>
      <property name="text">
        <string>计算</string>
      </property>
    </action>
    <action name="actionPrev">
      <property name="icon">
        <iconset resource="GlobalRes.qrc">
          <normaloff>:/Image/left.png</normaloff>:/Image/left.png
        </iconset>
      </property>
      <property name="text">
        <string>上一步</string>
      </property>
    </action>
    <action name="actionNext">
      <property name="icon">
        <iconset resource="GlobalRes.qrc">
          <normaloff>:/Image/right.png</normaloff>:/Image/right.png
        </iconset>
      </property>
      <property name="text">
        <string>下一步</string>
      </property>
    </action>
    <action name="actionConfig">
      <property name="icon">
        <iconset resource="GlobalRes.qrc">
          <normaloff>:/Image/gear.png</normaloff>:/Image/gear.png
        </iconset>
      </property>
      <property name="text">
        <string>设置</string>
      </property>
    </action>
    <action name="actionPlay">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="icon">
        <iconset resource="GlobalRes.qrc">
          <normaloff>:/Image/play.png</normaloff>:/Image/play.png
        </iconset>
      </property>
      <property name="text">
        <string>连续播放</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
    <include location="GlobalRes.qrc"/>
  </resources>
  <connections/>
</ui>
